toolkit/tools/internal/pkgjson/pkgjson.go

```diff
@@ -73,11 +74,54 @@ type Package struct {
 	RunTests      bool          `json:"RunTests"`      // Should we run tests for this package.
 }
 
+//cmp func(a, b E) int
+
+func PackageLess(a, b PackageVer) bool {
+	if a.Name < b.Name {
+		return true
+	} else if a.Name == b.Name {
+		v1 := versioncompare.New(a.Version)
+		v2 := versioncompare.New(b.Version)
+		if v1.LT(v2) {
+			return true
+		}
+	}
+
+	return false
```

Comment:
Simplification idea:
```suggestion
	if a.Name == b.Name {
		v1 := versioncompare.New(a.Version)
		v2 := versioncompare.New(b.Version)
		return v1.LT(v2)
	}

	return a.Name < b.Name
```

Reply:
Done